components/Navigation.js

```diff
@@ -47,30 +49,39 @@ export default function Navigation({
       {!isNavigationOpen && (
         <>
           <StyledSearchMenu onClick={toggleSearchBar}>
-          <Image src={"/icons/search.png"} width={35}  height={35} alt= {"search icon"}/>
+            <Image
+              src={"/icons/search.png"}
+              width={35}
+              height={35}
+              alt={"search icon"}
+            />
           </StyledSearchMenu>
           <StyledNavigationMenu onClick={toggleNavigation}>
-          <Image src={"/icons/burger_menu.png"} width={35}  height={35} alt= {"burger menu icon"}/>
+            <Image
+              src={"/icons/burger_menu.png"}
+              width={35}
+              height={35}
+              alt={"burger menu icon"}
+            />
           </StyledNavigationMenu>
         </>
       )}
 
       {isNavigationOpen && (
         <StyledNavigationMenuBar>
           <StyledNavigationMenu onClick={toggleNavigation}>
-          <Image src={"/icons/cancel.png"} width={35}  height={35} alt= {"cancel icon"}/>
+            <Image
+              src={"/icons/cancel_light.png"}
+              width={35}
+              height={35}
+              alt={"cancel icon"}
+            />
           </StyledNavigationMenu>
           <StyledNavigationLink href="/" onClick={toggleNavigation}>
             HomePage
           </StyledNavigationLink>
-          <StyledNavigationLink href="" onClick={toggleAddModal}>
+          <StyledNavigationLink href="/" onClick={toggleAddModal}>
```

Comment:
As already mentioned in the 1on1, you could think about using a button element, because this approach is somewhat unusual.